packages/flare/python/cron_job_ingest_events.py

```diff
@@ -73,6 +71,10 @@ def main() -> None:
 
         next = get_next(app=app, tenant_id=tenant_id)
         start_date = get_start_date(app=app)
+        logger.debug(
+            f"Fetching Tenant:{tenant_id}, Next:{next}, Start Date:{start_date}"
```

Comment:
Here you can just use `f" Fetching {tenant_id=}` and it will automatically display with what you want.

Reply:
docs: https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging

Reply:
Nice 👍